odoo/addons/base/migrations/13.0.1.3/pre-migration.py

```diff
@@ -760,6 +760,15 @@ def rename_ir_module_category(env):
         old_row = env.cr.fetchone()
         if old_row:
             if new_row:
+                query = "SELECT parent_id FROM ir_module_category WHERE id = %s"
+                env.cr.execute(query, (new_row[0], ))
+                new_parent_row = env.cr.fetchone()
+                if new_parent_row and new_parent_row[0] == old_row[0]:
+                    # When we already have recursive categories with same name
+                    # (example: Manufacturing/Manufacturing), doing a merge let
+                    # the akward situation where the parent points to itself,
+                    # so we avoid it checking this condition
```

Comment:
```suggestion
                    # When we already have recursive categories, doing a 
                    # merge of a parent into one of its childs let
                    # the akward situation where the child points to itself,
                    # so we avoid it checking this condition
```
maybe you would like to rewrite it a bit.

Reply:
Not really, as it's important that `Manufacturing/Manufacturing` text appears as reference. The rest helps, but I think it's enough for not launching another CI. Your call BTW.

Reply:
ok, then merge

Reply:
Thanks for your comprehension.